<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCreateWorkOrder</class>
 <widget class="QDialog" name="DialogCreateWorkOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>490</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新建工单</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1,1,1,0" columnstretch="1,3">
     <item row="6" column="0">
      <widget class="QLabel" name="labelPlanProductionQuantity">
       <property name="text">
        <string>计划生产数量:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxCraftID"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCraftID">
       <property name="text">
        <string>工艺编号：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelOrderID">
       <property name="text">
        <string>工单编号：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="dateEditPlanDate">
       <property name="date">
        <date>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editProductID">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelProductID">
       <property name="text">
        <string>产品编号：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelPlanDate">
       <property name="text">
        <string>计划生产日期:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelProductName">
       <property name="text">
        <string>产品名称：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editProductName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editOrderID"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editProductionBatch"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelProductionBatch">
       <property name="text">
        <string>生产批次：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editPlanProductionQuantity"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="editPlanInspectionQuantity"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelPlanInspectionQuantity">
       <property name="text">
        <string>计划抽检数量:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QTextEdit" name="editDescription"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelDescription">
       <property name="text">
        <string>描述:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAccept">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReject">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
